databases/business-sync/fragment-phone-verification-carrier-blacklist.js

```diff
@@ -0,0 +1,24 @@
+{
+  channels: function(doc, oldDoc) {
+    // Only staff/service users can create, replace or delete payment attempts to prevent regular users from tampering
+    return {
+      view: staffChannel,
+      write: staffChannel
+    };
+  },
+  typeFilter: function(doc, oldDoc) {
+    return /^phoneVerificationCarrierBlacklist$/.test(doc._id);
+  },
+  propertyValidators: {
+    blacklist: {
+      // The ID of the business with which the payment attempt is associated
+      type: 'array',
+      required: true,
+      minimumLength: 1,
```

Comment:
Should we make min length 0 to allow empty blacklist?

Reply:
Hmm, I think I would rather have this guard against an accidental empty blacklist being posted - the document can always be deleted, or the feature flag turned off, if we wanted to disable the blacklist entirely.

Reply:
👍